### ISSUE TITLE
Nikhil Pawar

### ISSUE THREAD

**Issue by aflies1** [promazo/Sourcing#166]
Summer internship for 3 months - IT analyst (PS business solutions) Heavly invested in NY state Depts. Mix between IT and Data Science, analyxed various data sets, finding KPIs.

After graduation? last semester - focused on working with data, analyst/management/science - applying ML techniques.  Sees himself as a full time DS in the next few years.

Has worked on a customer retention project with churn for a XYZ bank, had worked on a similar project last semester.

Python - has been working with it for 3 years, has used it extensively for the last 2 years. Would rank himself 7.5 on a scale of 10, understanding where and how to use it. 

SQL - Has been using SQL for over 3 years, queries, functions..... sounds pretty competitent

ML- has used R/Python, rapidminore, IBM watson (NLP as well) Xgboost? has used that algo before for some cluster analyst, used canon and random trees (projects throughout his master)

NLP - well versed in IBM watson - haworked with the stacks and has done a lot of different analysts with the models, but would be interested in the type of tools he would be using.

Schedule - graduation date of Dec 2022, will be officially graduation in May 23 - After dec. can take on full time work. As far as he knows he'd be able to use CPT through May? 18 hours per week? that would not be an issue, would be fine.

**Comment by aflies1** [comment]
Resume; https://drive.google.com/drive/u/0/folders/1CghJp1tRNhJdEINJSsjckIC2NiJejdE4

**Comment by ldaudet** [comment]
Nikhil R Pawar

Comprehension was difficult to me, in part because of a poor mic, in part becasue of the accent, but it could me a me thing, being english as a second language.

The two professional experience that he has listed, are much more computer science oriented than data science oriented. There was some light EDA but that's about it.

The laudry live project was more related to business acumen. He pitched the idea, and got a system to help people select a time to do their laundry, and avoid too long wait times.

The save a life blood donation project, they made the blueprint for an app, but did not actually code anything.

For the flight data analysis, he looked at a huge amount of data in csv file, from a bunch of airlines, and they were tasked to find what airlines at the most delays. Used IBM watson for it, which is funny to me, because I thought it was supposed to be tailored for medical use.

Has done some classification modeling, but no time series, or forecasting.

TL;DR: seems a bit green, didn't have much to talk about during the interview.
